cni-plugins/sriov-passthrough-cni/README.md

```diff
@@ -0,0 +1,29 @@
+# SR-IOV passthrough CNI
```

Comment:
I miss one information. If I make a change here, how do I propagate it to our cluster?